image-counter-lambda/artifact.json

```diff
@@ -0,0 +1,11 @@
+{
+  "projectName": "media-service::grid::image-counter-lambda",
+  "vcsURL": "your github url",
```

Comment:
Should this be https://github.com/guardian/ ?

Reply:
@AWare I'm happy to +1 this once that's added, keen to get this through!

Reply:
done, this'll also  need a +1 on https://github.com/guardian/editorial-tools-platform/pull/363